docs/guide/contributing.rst

```diff
@@ -31,37 +31,35 @@ We especially need contributions in these areas:
 * Developer tools or docs-as-code toolchains and workflows.
 * High-level discussions of common tools. 
 
-For help finding where a new set of topics best fit into the rest of 
+For help finding the best fit for new topics in the rest of 
 the guide, `ask in Slack`_, or `file a GitHub issue`_.
 
-Guidelines
------------
+Contribution guidelines
+-------------------------
 
-Follow these guidelines for discussing tools: 
+General guidelines include:
 
-* Ensure your content is of interest to a general audience.
-* Avoid advocacy and plugs about your favorite toolchain, even if it's open source.
-* Present specific use cases, how problems were solved, and what worked or didn't work well. 
-* Consider to first `file a GitHub issue`_ or contact one of the guide editors at guide@writethedocs.org.
+* Use a friendly and encouraging tone. It's a good way to write docs, so practice with a guide at Write the Docs!
+
+* Attribute links and mention why linked resources are useful.
+
+* Although some files in the guide are written in `Markdown`_, we prefer `reStructuredText (.rst)`_.
 
-Keep content focused on general principles and best practices instead of arguing over minor 
-points at the expense of major principles of clarity and communication. Avoid preferential 
-editing that only offers your personal preferences for dealing with a particular writing 
-situation. For example, if you recommend a particular word choice, tell the audience why it 
-matters.
+* Focus content on general principles and best practices instead of arguing over minor points that impede clarity and communication. 
 
-Use a friendly and encouraging tone. It's a good way to write docs in general, so practice 
-with docs about docs!
+* Avoid preferential editing that only offers your personal preferences for dealing with a particular writing situation. For example, if you recommend a word choice, tell the audience why it matters.
 
-When linking to resources, attribute the links appropriately and mention why the resources 
-are useful.
+Additionally, follow these guidelines when discussing tools: 
+
+* Ensure your content is of interest to a general audience.
+* Avoid advocacy and plugs about your favorite toolchain, even if it's open source.
+* Present specific use cases, how problems were solved, and what worked or didn't work well. 
+* Consider to first `file a GitHub issue`_ or contact a guide editor at guide@writethedocs.org.
 
-Some files in the guide are written in `Markdown`_, and some 
-are written in `reStructured Text (rST)`. We prefer rST 
-although either format is acceptable.
+.. _prerequisites:
 
-Prerequisites
---------------
+Prerequisites for contributing
+-------------------------------
```

Comment:
I'm not sure I understand the need to make this longer. Could you explain? `Prerequisites` (or more informally `Before you start`) is a common heading in how-to guides, I think, and is usually clear enough on it's own, I would hope.

Reply:
Especially true give that these aren't prerequisites for contributing, but for contributing in a certain way.

Reply:
Since, as you mentioned, these are prerequisites for contributing a certain way, should we actually clarify even further then? Something like "Prerequisites for contributing via git"? Otherwise, as you mentioned in another comment that the structure of the content should make the context of the prerequisites obvious, we should just move the prerequisites section under the "Update a guide in Write the Docs via a pull request" section.

Reply:
Also, we should be cognizant that many individuals will come to know about a section of a doc via a search on the website and that we should err towards extra context most of the time (plus, SEO will care about how we name our headers such that we may need to be more redundant despite the context of the article (not that either of these arguments matter for this particular situation).  Just letting the group know where I'm coming from.

![Screenshot 2024-07-11 8 18 58 AM](https://github.com/writethedocs/www/assets/64424206/4dd995ea-366e-4b50-9900-ef996a8af303)

Reply:
I don't think that someone will be searching for "prerequisites" but itself. I feel like SEO might actually be an argument for not having additional context in there. Because people shouldn't be coming to that heading on its own, just within the context.

Take an example from [Google's cloud docs[(https://cloud.google.com/run/docs/quickstarts/build-and-deploy/deploy-python-service) (which I assume would have some insight into the SEO advantages of headings). They consistently use `Before you begin`.

It's not something I feel would be blocking either way, just saying. 🙂

Reply:
I know they won't search for "prerequisites" by itself. It is just an example to show one of the many reasons why we need to be careful about headers in general. 

The biggest takeaway from the example you provided from Google is that we shouldn't be using formal, complicated language like "prerequisites" at all and should use simpler text like "before you begin." Do you agree? Thanks for sharing the example!

Reply:
Yeah, I like it better. I remembered it as `Before you start` in my first message in this thread, but really either way is fine.